command-line/command-line-interface.md

```diff
@@ -26,6 +26,34 @@ $ cd Downloads/
 - Go to www.npmjs.com/package/express, which is the Node Package Manager page for Express.js.
 - Everything in the block after the "$" is the command that you would type into the Command Line in order to install Express.js.
 
+### Set Up zsh
+First, if you purchased a Mac with the latest operating system, **Catalina**, *zsh* is already the default shell. Be sure to check by typing the following to your command line:
+
+```
+zsh --version
+```
+
+If you currently own a Mac and just recently upgraded your operating system to Catalina, please type the following line to switch the default shell from *Bash* to *zsh*:
+
+```
+chsh -s $(which zsh)
+```
+
+After you changed to zsh, log out of your Terminal and log back in. To check to be sure, type the following:
+
+```
+$SHELL --version
+```
+
+The result should give you which shell the Terminal is set up and the version of the shell.
+
+#### Configure your Terminal with Oh My ZSH (Optional)
```

Comment:
Please move this to Challenge below

Reply:
I fixed this months ago on the `command-line-interface.md` file. Did you mean the Oh My ZSH section is part of the Challenge section?